core/src/themes/themes.interfaces.ts

```diff
@@ -17,6 +17,9 @@ export type BaseTheme = {
     [key: string]: string;
   };
 
+  // TODO(FW-6864): Remove once IonToolbar themes are added
+  toolbar?: any;
+
```

Comment:
Tests fail when `--ion-toolbar-background` is not set in certain cases like `md/dark`. Since the goal of this PR is to enable the palettes, I didn't think create this variable under the components since as of now it would have rendered it as `--ion-components-ion-toolbar-background`. I would have had to either update the generate function to not add the `--ion-components` suffix or update all the `--ion-toolbar-background` to the new variable if I wanted to do it properly. That would have been outside of the scope so this approach was taken along with adding a TODO to come back.